src/oas2/transformers/params.ts

```diff
@@ -157,11 +201,9 @@ function buildContentFromParameter(
     'multipleOf',
   );
 
-  return {
-    schema: pickBy(
-      defaults<Schema, Schema>(schema, {
-        minLength: 'allowEmptyValue' in param && param.allowEmptyValue ? 1 : undefined,
-      }),
-    ),
-  };
+  if ('allowEmptyValue' in param && param.allowEmptyValue) {
```

Comment:
Yeah, I admit the readability of this line of code is way better than the previos one 👍